### ISSUE TITLE
curl throws error in terraform plan action.

### ISSUE THREAD

**Issue by tylersuggs** [hashicorp/terraform-github-actions#58]
We are receiving the follow error when running `hashicorp/terraform-github-actions/plan@v0.3.7`. This is preceded by the `init` and `validate` actions of the same version that do not error.
```
Error relocating /usr/bin/curl: curl_multi_poll: symbol not found
##[error]Docker run failed with exit code 127
```
According to the output, the `plan` command succeeds, but the curl error causes a false-negative for the workflow.

If more information is required, I will provide what I can.

**Comment by jeffatgametime** [comment]
Ditto. This is only a problem with the new workflow syntax (yaml) from what I've experienced. the HCL version of the workflow works.

I converted from HCL to yaml using [this tool](https://github.com/actions/migrate/releases/) suggested by the github action docs.

**Comment by jeffatgametime** [comment]
This PR would fix this https://github.com/hashicorp/terraform-github-actions/pull/59

**Comment by sudomateo** [comment]
Fixed in #62.
